spec/index.html

```diff
@@ -315,28 +315,27 @@ <h3>RDF Vocabularies and Namespace IRIs</h3>
     <h3>Triple Terms and Reification</h3>
 
     <p>A <a>triple term</a> is an [=RDF triple=] used as an [=RDF term=]
-      in the [=object=] position of another triple.</p>
-
-    <p>A <a>triple term</a> is not necessarily asserted, allowing
-      statements to be made about other statements that may not be
-      asserted within an <a>RDF graph</a>.
-      This allows statements to be made about relationships that may be contradictory.
-      For a <a>triple term</a> to be asserted,
-      it must also appear in a graph as an <a>asserted triple</a>.</p>
-
-    <p>A <a>triple term</a> can be used as the object of a <a>triple</a> with the predicate <code>rdf:reifies</code>;
-      such a <a>triple</a> is then a <dfn>reifying triple</dfn>.
-      The <a>subject</a> of that <a>triple</a> is called a <dfn>reifier</dfn>.
-      Assertions on the <a>triple term</a> are made using the <a>reifier</a>.
-      A concrete syntax may provide a special notation for specifying <a>reifying triples</a>.</p>
-
-    <p class="note">Using the <a>reifier</a> as an indirect way of referencing a <a>triple term</a>
-      allows multiple groups of assertions to be separated from each other,
-      which might be necessary if the same <a>triple term</a> was derived from different sources.
-      Assertions are always made using the <a>reifier</a>, which can be the <a>subject</a> or <a>object</a>
-      of different triples.
-      Concrete syntaxes, such as Turtle [[RDF12-TURTLE]],
-      may have shortcuts for capturing a <a>triple term</a> with its <a>reifier</a>.</p>
+      in another triple. This use is a reference to a <a>proposition</a>. For this
+      triple to also be asserted, it must appear in a graph as an <a>asserted
+      triple</a>. This allows for statements to be made about statements
```

Comment:
```suggestion
      in another triple. This use is a reference to a <a>proposition</a>. For
      the triple term to also be asserted, it must appear in a graph as an <a>asserted
      triple</a>. This allows for statements to be made about statements
```
"this triple" is highly ambiguous in that sentence! ("that triple" would probably be more accurate, but still too ambiguous to my taste).